biorbd_optim/limits/path_conditions.py

```diff
@@ -9,7 +9,7 @@
 class PathCondition(np.ndarray):
```

Comment:
Please provide unit tests for all the different slicing. 
Also, you should raise an error if a slice is asked for a Custom already sliced